src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -562,4 +634,19 @@ PackageDetailTab.contextTypes = {
   router: routerShape
 };
 
+const dependency = cosmosPackage => (cosmosPackage.manager || {}).packageName;
+
+const unresolvedDependencyMessage = dependency => (
+  <div>
+    <Icon shape={SystemIcons.CircleInformation} size={iconSizeXs} />{" "}
+    <Trans>
+      This service cannot run without the "
+      <Link to={`/catalog/packages/${dependency}`}>
+        {StringUtil.capitalize(dependency)}
```

Comment:
Worth noting that this is not a foolproof method of capitalization -- Thinking of "Dcos" here

I think it's better to leave it unchanged since I mentioned this to design already and the package names are in lowercase in the catalog.

Reply:
Also, I can't seem to get the Tooltip to dismiss. I thought these were supposed to be magically dismissed by ui-kit.

Reply:
@brandonc - this Tooltip is from `reactjs-components`.

I can't get it to dismiss either, but it's dismissing just fine everywhere else.

Reply:
interesting: the tooltip does not trigger `onMouseLeave` from a disabled input. 

i changed the implementation to add a `disabled`-class instead of using the attribute. this way it seems to work just fine.

this also fixes the tooltip behaviour for the other cases the button is disabled!